lms/static/sass/views/_login-register.scss

```diff
@@ -415,10 +435,6 @@ $sm-btn-linkedin: #0077b5;
                 border: 1px solid #A5382B;
                 color: white;
             }
-
-            &:hover {
-                box-shadow: 0 2px 1px 0 #8D3024;
-            }
```

Comment:
Hmm, on http://sso.sandbox.opencraft.com/login they do... But it is inconsistent. I personally think this is better.